docs/source/conf.py

```diff
@@ -0,0 +1,530 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+import typing as t
+import inspect
+import os
+import sys
+import pathlib
+import functools
+from subprocess import check_output
+# from sphinx.ext.autosummary import Autosummary
+# from sphinx.ext.autosummary import get_documenter
+# from docutils.parsers.rst import directives
+# from sphinx.util.inspect import safe_getattr
+# import re
+
+
+CURRENT_DIR = pathlib.Path(__file__).parent
+PROJECT_DIR = CURRENT_DIR.parent.parent
+
+
+sys.path.insert(0, str(PROJECT_DIR.absolute()))
+
+
+import deepchecks.version
```

Comment:
I can't import this locally, can you remove it from here, or add it to the package?

Reply:
do you mean that you get ImportError when trying to import `deepchecks.version`?

Reply:
yup